server/data/data-jdbc/src/main/java/org/apache/james/mailrepository/jdbc/JDBCMailRepository.java

```diff
@@ -380,14 +361,12 @@ protected void checkJdbcAttributesSupport(DatabaseMetaData dbMetaData) throws SQ
         }
         if (hasUpdateMessageAttributesSQL && hasRetrieveMessageAttributesSQL) {
             jdbcMailAttributesReady = true;
-            if (LOGGER.isInfoEnabled()) {
-                logBuffer.append("JDBC Mail Attributes support ready.");
-                LOGGER.info(logBuffer.toString());
-            }
+            logBuffer.append("JDBC Mail Attributes support ready.");
+            LOGGER.info("{}", logBuffer);
```

Comment:
Oups?

Reply:
This is intentional; its purpose is to avoid creating a String of the contents of `logBuffer` if logging at the info level is not enabled.